src/harbor.py

```diff
@@ -155,6 +168,27 @@ async def main() -> None:
         print("Skipping syncing retention policies")
 
 
+async def sync_garbage_collection_schedule(
+    garbage_collection_schedule: Schedule
+) -> None:
+    try:
+        gc_schedule = await client.get_gc_schedule()
```

Comment:
Should we do the `get_gc_schedule` in the same way as on the purge_job_schedule (in the if statement)?
It could be confusing, that we just need the `gc_schedule` object for the try/except check.

Reply:
Updated it so there is no `gc_schedule` object. Also changed to try&except in both functions to make it consistent.